src/main/java/mate/academy/dao/GenericDao.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+
+public interface GenericDao<T, I> {
+    T add(T entity);
+
+    Optional<T> get(I id);
```

Comment:
if you return `Optional` you should name method as `findById`

Reply:
Name of the method was taken from task, but I decide that better to return Optional then null.